app/src/ui/diff/diff-options.tsx

```diff
@@ -139,9 +150,6 @@ export class DiffOptions extends React.Component<
   }
 
   private renderHideWhitespaceChanges() {
-    if (this.props.hideWhitespaceChanges === undefined) {
-      return null
-    }
```

Comment:
I think we still need some kind of check, using also the feature flag, to prevent this checkbox to be shown in the Changes tab when the feature flag is not enabled. Is that right? 🤔